apps/web-giddh/src/app/shared/amount-field/amount-field.component.ts

```diff
@@ -12,7 +12,8 @@ export class AmountFieldComponent implements OnInit, OnChanges {
     @Input() public currencySymbol: string;
     @Input() public currencyCode: string;
     @Input() public ratePrecision: boolean = true;
-
+    @Input() public bankType: { type: string };
+    @Input() public convertedCurrencyCode: string;
```

Comment:
pass convertedCurrencyCode in currencyCode input instead of using it as separate input

Reply:
Removed convertedCurrencyCode, it is not using anywhere as a variable